### PR TITLE
allows to remove stealing objectives per map

### PR DESCRIPTION
**Pull request by cumoverme**
I have no idea what I'm doing

### CODE REVIEW

File: maps/_map.dm [hunk 1]
```diff
@@ -37,6 +37,9 @@
 	var/tDomeY = 0
 	var/tDomeZ = 0
 
+	//List for traitor items which are not in the map
+	var/unavailable_items[]
```
Comment (PJB3005) [inline maps/_map.dm]: Make this variable a `var/list/...`

**Comment by Shadowmech88** [comment]
RPDs can be made at an autolathe.
Is there no autolathe on Ministation?

**Comment by Imagebard** [comment]
>cumoverme

Good job winning those bomberman rounds.

**Comment by xpcybic** [comment]
@Shadowmech88 There's one in cargo and a board for another in R&D, but engineering already has one RPD as well

File: code/game/gamemodes/objective.dm [hunk 2]
```diff
@@ -536,6 +536,9 @@ var/list/potential_theft_objectives=list(
 		var/pickedObjective = pick(possibleObjectives)
 		var/datum/theft_objective/objective = new pickedObjective
 
+		if(objective.typepath in map.unavailable_items)
```
Comment (ComicIronic) [inline code/game/gamemodes/objective.dm:539]: Remind me about DM behaviour - does this throw for null?

Comment (PJB3005) [inline code/game/gamemodes/objective.dm:539]: Nope

**Comment by ComicIronic** [comment]
Why are RPDs still steal objectives?

**Comment by PJB3005** [comment]
Because it's oldcode and nobody bothered to remove them?

**Comment by Imagebard** [comment]
A lot of shit on the steal objectives list are easy to get.
And nobody bothers updating them because a lot of shit on the steal objectives list are easy to get.

**Comment by ComicIronic** [comment]
First PR to fix the steal objectives list gets insta merge and promotion to Collaborator.
If someone in the thread complains, though, you're getting demoted and banned.

**Comment by Imagebard** [comment]
ComicIronic what are you doing here

**Comment by gbasood** [comment]
>comic trying to promote people to collab

what the fuck are you doing

**Comment by 9600bauds** [comment]
I don't think this needs a changelog until it's actually used or until ministation goes in rotation

**Comment by gbasood** [comment]
@9600bauds I was gonna say that this is the sort of thing players should know is possible but yeah maybe not actually

**Comment by ComicIronic** [comment]
Who's going to remember to changelog it when Mini comes back?

**Comment by clusterfack** [comment]
Tell them how to changelog then I guess, you still here @cumoverme ?

**Comment by clusterfack** [comment]
Nah fuck it

**Comment by Imagebard** [comment]
Clusterfack don't you think your merging has become disorganized lately

**Comment by clusterfack** [comment]
Yes I'm an absolute madman

**Pull request merged**